bower-publish.sh

```diff
@@ -7,6 +9,8 @@ if [ $CURRENT_BRANCH != 'master' ] ; then
   exit 0
 fi
 
+CHOSEN_VERSION=`git tag -l "*.*.*" | sed 's/^v//' | sort -V | tail -1`
```

Comment:
`git tag` has support for sorting, but due to the inconsistency in tags (with and without v prefix) that isn't working out either.. (cc #2450)

```
$ git tag --sort=v:refname
0.9
0.12.1
0.13.0
0.14.0
1.0.0
1.4.0
1.4.2
semver
v0.9
v0.9.1
v0.9.2
v0.9.3
v0.9.4
v0.9.5
v0.9.6
v0.9.7
v0.9.8
v0.9.9
v0.9.10
v0.9.11
v0.9.12
v0.9.13
v0.9.14
v0.9.15
v0.10.0
v0.10.1
v0.11.1
v0.12.0
v1.0.0
v1.1.0
v1.1.1
v1.1.2
v1.2.0
v1.3.0
v1.4.0
v1.4.1
```